server/coordinator.go

```diff
@@ -184,6 +197,16 @@ func (c *coordinator) addOperator(op Operator) bool {
 		c.operators[regionID] = op
 		return true
 	}
+	if op.GetResourceKind() == priorityKind {
+		o, ok := c.operators[regionID]
+		if ok && o.GetResourceKind() == priorityKind {
+			return false
+		}
+		c.limiter.addOperator(op)
+		c.operators[regionID] = op
```

Comment:
If there are an another type operator for this region, it will be dropped by the new operator?